src/components/forms/PlanForm/tests/fixtures.ts

```diff
@@ -629,6 +703,32 @@ export const extractedActivitiesFromForms = [
     timingPeriodEnd: timingPeriodEnd.toISOString(),
     timingPeriodStart: timingPeriodStart.toISOString(),
   },
+  {
+    actionCode: 'MDA Adverse Event(s)',
+    actionDescription: 'Report any adverse events from medication',
+    actionIdentifier: '',
```

Comment:
Do we need the actionIdentifier to be blank?

Reply:
My thought is `actionIdentifier` comes from the API. Even when defining the actions on [settings](https://github.com/onaio/reveal-frontend/blob/14bfb21d52016673392fc6e18a7d5d5b33ac37f5/src/configs/settings.ts#L354) it is left blank